### PR TITLE
Ignore other volume types by default

### PR DESCRIPTION
**Pull request by WanzenBug**
See #18 for motivation.

This currently only works for the most common other volume types. There are a bunch of other volumes which might work, depending on their read-only flag.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Exempt POds that are attached to other types of storage by default, unless the volumes are known to be safe (such as
```
Comment (kvaps) [inline CHANGELOG.md:10]:
```suggestion
- Exempt Pods that are attached to other types of storage by default, unless the volumes are known to be safe (such as
```

**Review by kvaps — COMMENTED** [review]
Would this work in case if ConfigMap is mounted with `readOnly: false`?

```yaml
containers:
- volumeMounts:
  - mountPath: /config
    name: config
    readOnly: false
volumes:
- name: config
  configMap:
    name: test
```

**Comment by WanzenBug** [comment]
> Would this work in case if ConfigMap is mounted with `readOnly: false`?

Hmm, I'm not familiar with what exactly this option changes. When the pod is deleted and a replacement is started, k8s will just recreate the original state as specified in the API resources, right?

**Comment by kvaps** [comment]
Hmm, you're righ.
This will not gonna work starting from k8s v1.9 and from v1.11 with feature gate `ReadOnlyAPIDataVolumes` disabled

- https://github.com/kubernetes/kubernetes/issues/62099
- https://github.com/kubernetes/kubernetes/pull/60258

**Review by kvaps — APPROVED** [review]

**Pull request merged**